src/OrleansCodeGenerator/GrainReferenceGenerator.cs

```diff
@@ -61,12 +61,13 @@ internal static TypeDeclarationSyntax GenerateClass(Type grainType, Action<Type>
             var attributes = SF.AttributeList()
                 .AddAttributes(
                     CodeGeneratorCommon.GetGeneratedCodeAttributeSyntax(),
-#if !NETSTANDARD
-                    // we could add Serializable attribute if we want, but that would require that the target
-                    // project depends on System.Runtime.Serialization.Formatters, which is currently in preview
+
                     SF.Attribute(typeof(SerializableAttribute).GetNameSyntax()),
+#if !NETSTANDARD_TODO
+                    //ExcludeFromCodeCoverageAttribute became an internal class in netstandard
                     SF.Attribute(typeof(ExcludeFromCodeCoverageAttribute).GetNameSyntax()),
 #endif
+
```

Comment:
I just stopped by since you assigned to me,  but since I can't download locally or even run/check functionals, I'll leave it to someone else on the team. The PR looks good to me as far as I can tell from github